pkg/apis/core/validation/validation_test.go

```diff
@@ -18619,6 +18619,54 @@ func TestValidateServiceUpdate(t *testing.T) {
 	}
 }
 
+func TestValidatePodResourceNames(t *testing.T) {
+	table := []struct {
+		input   core.ResourceName
+		success bool
+		expect  string
```

Comment:
I don't see `expect` being used in any of the test case.

Reply:
Fixed, had copied the test code from the function below this function (which requires refactoring).